components/dialog/dialog-mixin.js

```diff
@@ -35,6 +37,7 @@ export const DialogMixin = superclass => class extends RtlMixin(superclass) {
 			 * The optional title for the dialog
 			 */
 			titleText: { type: String, attribute: 'title-text' },
+			_autoSize: { type: Boolean },
```

Comment:
Probably want `attribute: false` so that nothing gets reflected back & forth?

Reply:
It should not get reflected, but yeah, we're not expecting consumer to set this.  Will add.